### ISSUE TITLE
[FEA] abs function for dataframe class

### ISSUE THREAD

**Issue by shridharathi** [rapidsai/cudf#8614]
**Is your feature request related to a problem? Please describe.**
I wish I could use cuDF could support an absolute value function which calculates the entry-wise absolute value for a cudf dataFrame.

**Describe the solution you'd like**
A single _unaryop("abs") call

**Describe alternatives you've considered**
Iterating through each column and calling abs on each as a series (abs is supported for series currently)

**Comment by harrism** [comment]
@beckernick libcudf unary ops supports the ABS operator. However unary ops is a column operation, not a table operation. Should be easy enough at the Python level to iterate over columns and call the libcudf unary op.

**Comment by beckernick** [comment]
Agreed. Python layer has the machinery for this already, but it's one op per column as you noted.

**Comment by vyasr** [comment]
The `__abs__` operator is already implemented for `DataFrame` and works as expected:

```
>>> import cudf
>>> df = cudf.DataFrame({'a': [1, 2, 3], 'b': [-4, -5, 6]})
>>> abs(df)
   a  b
0  1  4
1  2  5
2  3  6
```

Would that be sufficient for your usage for now? I plan to also enable `df.abs()`, I'm just waiting on #8598 to get merged first to minimize conflicts.

**Comment by beckernick** [comment]
Looks like @shridharathi had a WIP PR but closed it. @shridharathi is currently exploring some other functionality, so I think that timeline might work well. What do you think @shridharathi ?

**Comment by shridharathi** [comment]
@beckernick Yes probably relying on #8598 would minimize the redundancies with some of the other functions as well

**Comment by vyasr** [comment]
@isVoid when you consolidate the methods that we discussed today into Frame that should automatically resolve this issue.
